### ISSUE TITLE
Argument #1: Expected object but got string

### ISSUE THREAD

**Issue by alexlrobertson** [npm/npm#8931]
```
ch-nb-467:up-ui-core arobertson$ npm install eslint-plugin-angular
npm ERR! Darwin 14.4.0
npm ERR! argv "node" "/Users/arobertson/.nvm/versions/node/v0.12.7/bin/npm" "install" "eslint-plugin-angular"
npm ERR! node v0.12.7
npm ERR! npm  v3.1.1
npm ERR! code EINVALIDTYPE

npm ERR! typeerror Error: Argument #1: Expected object but got string
npm ERR! typeerror     at resolveWithNewModule (/Users/arobertson/.nvm/versions/node/v0.12.7/lib/node_modules/npm/lib/install/deps.js:375:3)
npm ERR! typeerror     at /Users/arobertson/.nvm/versions/node/v0.12.7/lib/node_modules/npm/lib/install/deps.js:163:5
npm ERR! typeerror     at /Users/arobertson/.nvm/versions/node/v0.12.7/lib/node_modules/npm/node_modules/slide/lib/async-map.js:52:35
npm ERR! typeerror     at Array.forEach (native)
npm ERR! typeerror     at /Users/arobertson/.nvm/versions/node/v0.12.7/lib/node_modules/npm/node_modules/slide/lib/async-map.js:52:11
npm ERR! typeerror     at Array.forEach (native)
npm ERR! typeerror     at asyncMap (/Users/arobertson/.nvm/versions/node/v0.12.7/lib/node_modules/npm/node_modules/slide/lib/async-map.js:51:8)
npm ERR! typeerror     at exports.loadRequestedDeps (/Users/arobertson/.nvm/versions/node/v0.12.7/lib/node_modules/npm/lib/install/deps.js:158:3)
npm ERR! typeerror     at Array.<anonymous> (/Users/arobertson/.nvm/versions/node/v0.12.7/lib/node_modules/npm/node_modules/slide/lib/bind-actor.js:15:8)
npm ERR! typeerror     at LOOP (/Users/arobertson/.nvm/versions/node/v0.12.7/lib/node_modules/npm/node_modules/slide/lib/chain.js:15:14)
npm ERR! typeerror This is an error with npm itself. Please report this error at:
npm ERR! typeerror     <http://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/arobertson/WebstormProjects/up-ui-core/npm-debug.log
```

**Comment by iarna** [comment]
Hi, thanks for reporting this! I'm closing this as a duplicate of #8893 and this should be fixed with a new release later today #8922

**Comment by alexlrobertson** [comment]
Thanks!
